addons/test_sale_product_configurators/static/tests/tours/product_configurator_optional_products_ui.js

```diff
@@ -19,45 +19,26 @@ registry.category("web_tour.tours").add('sale_product_configurator_optional_prod
 }, {
     trigger: 'ul.ui-autocomplete a:contains("Customizable Desk (TEST)")',
 }, {
-    trigger: 'tr:has(.td-product_name:contains("Office Chair Black")) .js_add',
+    trigger: 'tr:has(.o_sale_product_configurator_name:contains("Office Chair Black")) button:has(i.fa-shopping-cart)',
 }, {
-    trigger: 'tr:has(.td-product_name:contains("Customizable Desk")) .fa-plus'
+    trigger: 'tr:has(.o_sale_product_configurator_name:contains("Customizable Desk")) button:has(i.fa-plus)'
 }, {
-    trigger: 'tr:has(.td-product_name:contains("Chair floor protection")) .js_add',
+    trigger: 'tr:has(.o_sale_product_configurator_name:contains("Chair floor protection")) button:has(i.fa-shopping-cart)',
 }, {
-    content: 'Is below its parent 1',
-    trigger: 'tr:has(.td-product_name:contains("Office Chair Black")) + tr:has(.td-product_name:contains("Chair floor protection"))'
+    trigger: 'tr:has(.o_sale_product_configurator_name:contains("Conference Chair")) button:has(i.fa-shopping-cart)',
 }, {
-    trigger: 'tr:has(.td-product_name:contains("Conference Chair")) .js_add',
-}, {
-    trigger: 'tr:has(.td-product_name:contains("Conference Chair")) .fa-minus'
-}, {
```

Comment:
That was lost, but it's still interesting to add something (maybe made sure it's added), then delete it (and also check)